.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Test
+on:
+  push:
+  pull_request:
```

Comment:
Do we not need to specify the branch (main)?

Reply:
no, it's better not to